integration/base/src/main/java/com/inrupt/client/integration/base/ApplicationRequestMetadataScenarios.java

```diff
@@ -0,0 +1,298 @@
+            .path(State.PRIVATE_RESOURCE_PATH + "-auth-test-" + UUID.randomUUID() + "/")
+            .build();
+
+        Utils.createContainer(authenticatedClient, privateContainerURI);
+
+        LOGGER.info("Integration Test Pod Host: [{}]", URI.create(podUrl).getHost());
+        if (!featureIsActive()) {
+            LOGGER.info("ApplicationRequestMetadataScenarios are skipped, feature not active");
+        }
+    }
+
+    @AfterAll
+    static void teardown() {
+        //cleanup pod
+        Utils.deleteContentsRecursively(authenticatedClient, publicContainerURI);
+        Utils.deleteContentsRecursively(authenticatedClient, privateContainerURI);
+    }
+
+    @ParameterizedTest
+    @MethodSource("provideSessions")
+    @DisplayName(" " +
+            "Request and response headers match for a successful authenticated request")
+    void requestResponseMatchOnAuthRequestLowLevelClientTest(final Session session) {
+        assumeTrue(featureIsActive());
+
+        LOGGER.info("Integration Test - Low level sync client - " +
+                "Request and response headers match for a successful authenticated request");
+
+        final SolidSyncClient authClient = SolidSyncClient.getClient().session(session);
+
+        final String resourceName = privateContainerURI + "e2e-test-application-metadata-" + UUID.randomUUID();
+        final String predicateName = "https://example.example/predicate";
+        final IRI booleanType = rdf.createIRI("http://www.w3.org/2001/XMLSchema#boolean");
+
+        final IRI subject = rdf.createIRI(resourceName);
+        final IRI predicate = rdf.createIRI(predicateName);
+        final Literal objectTrue = rdf.createLiteral("true", booleanType);
+
+        // Populate data for this resource
+        final Dataset dataset = rdf.createDataset();
+        dataset.add(null, subject, predicate, objectTrue);
+
+        final URI resourceUri = URI.create(resourceName);
+
+        final var headers = fillHeaders("aaaaaa");
+
+        // Create a new resource and check response headers
+        final Request.Builder reqBuilder = Request.newBuilder(resourceUri)
+                .header(Utils.CONTENT_TYPE, Utils.TEXT_TURTLE)
+                .PUT(Request.BodyPublishers.noBody());
+
+        //fill the configured headers
+        headers.forEach((key,value) -> {
+            reqBuilder.header(key, value.get(0));
+        });
+
+        final var req = reqBuilder.build();
+
+        final var res = authClient.send(req, Response.BodyHandlers.discarding());
+
+        assertTrue(Utils.isSuccessful(res.statusCode()));
+
+        for (String value: REQUEST_METADATA_HEADERS_THAT_PROPAGATE) {
+            assertEquals(headers.get(value), res.headers().allValues(value));
+        }
+    }
+
+    @ParameterizedTest
+    @MethodSource("provideSessions")
+    @DisplayName(" " +
+            "Request and response headers match for a successful authenticated request")
```

Comment:
The display name doesn't specify whether this is high or low-level client, I don't know if it should.

Reply:
The display name needs to reference a scenario from RDF, when I add documentation. So it should be the same scenario regardless what client.
What I should improve is the logging to mention the client type.